README.md

```diff
@@ -70,6 +70,18 @@ Since 1.3.0:
 * `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
 * `quarkus.tools.validation.value.severity`: Validation severity for property values for Quarkus/MicroProfile `*.properties` files. Default is `error`.
 
+### **Note for MicroProfile Rest Client properties**:
+
+Due to [this issue](https://github.com/redhat-developer/quarkus-ls/issues/203), the MP Rest property: `<mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority` reports an unknown error. 
+
+To avoid having this error, you must configure the following `settings.json`:
+
+```json
+"quarkus.tools.validation.unknown.excluded": [
+    "*/mp-rest/providers/*/priority"
```

Comment:
why don't we set it by default in the vscode-quarkus preferences?

Reply:
Because I dodn't know this feature :) Let me read doc to change my PR.

Reply:
@fbricon  fixed